schemas/workflow_runs/definitions.yaml

```diff
@@ -21,6 +21,9 @@ workflow_run_shared:
         type: string
         minLength: 1
         maxLength: 128
+    sdk_token:
```

Comment:
I guess this should be added in the `workflow_run_response` object, as it's not a field user can provide.

Reply:
done

Reply:
As discussed, this property should be in the `workflow_run_response` section but let's keep this new property aside for the time being.